<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>421</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="alpha_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="theta_label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="delta_label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hundredHertz_label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="twentymin_label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="fourtyfive_label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="userdesignated_label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="select_button">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>SELECT</string>
     </property>
    </widget>
    <widget class="QLabel" name="level_1">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>290</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>  1</string>
     </property>
    </widget>
    <widget class="QLabel" name="level_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>270</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>  2</string>
     </property>
    </widget>
    <widget class="QLabel" name="level_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>250</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>  3</string>
     </property>
    </widget>
    <widget class="QLabel" name="level_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>230</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>  4</string>
     </property>
    </widget>
    <widget class="QLabel" name="level_5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>  5</string>
     </property>
    </widget>
    <widget class="QLabel" name="level_6">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>  6</string>
     </property>
    </widget>
    <widget class="QLabel" name="level_7">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>  7</string>
     </property>
    </widget>
    <widget class="QLabel" name="level_8">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>  8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="down_button">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>230</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>DOWN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="up_button">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>170</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>UP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="power_button">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>230</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>POWER</string>
     </property>
    </widget>
    <widget class="QLabel" name="int_label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>     INT</string>
     </property>
    </widget>
    <widget class="QLabel" name="canada_label">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>270</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>CANADA</string>
     </property>
    </widget>
    <widget class="QLabel" name="mindalive_label">
     <property name="geometry">
      <rect>
       <x>343</x>
       <y>27</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>MIND ALIVE</string>
     </property>
    </widget>
    <widget class="QLabel" name="leftear_label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel</string>
     </property>
    </widget>
    <widget class="QLabel" name="oasispro_label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>100</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="topleft_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="bottomleft_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="power_light">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>280</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hundredHz_light">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="userdesignated_light">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="fourtyfivemin_light">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="twentymin_light">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="delta_light">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="theta_light">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="alpha_light">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="setDuration_button">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>16</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="setSession_button">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>16</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="userdes_comboBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>461</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>470</y>
      <width>82</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Left Ear</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>500</y>
      <width>82</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Right Ear</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>470</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wet earlobe (L)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>500</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wet earlobe (R)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recordSessionButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>470</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Record Session</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewDatabaseButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>470</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>View Session Logs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="replace_battery_button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>470</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Replace Battery</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
